presidio-anonymizer/presidio_anonymizer/operators/keep.py

```diff
@@ -1,14 +1,15 @@
 """Keeps the PII text unmodified."""
+from abc import ABC
 from typing import Dict
 
 from presidio_anonymizer.operators import Operator, OperatorType
 
 
-class Keep(Operator):
```

Comment:
I would suggest to keep `Keep` as it might be used by some users already, and this would potentially not be backward compatible

Reply:
Sure! Amended, keeping `Keep` and fixed testing